packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -472,52 +475,66 @@ export const Stackbox = () => {
                 );
               })}
             </div>
-            <div className="space-y-1">
-              <div className="flex flex-col border divide-y md:divide-y-0 md:flex-row rounded-2xl border-surface-50 md:divide-x divide-surface-50">
-                <div className="flex flex-col w-full py-2 pl-4 pr-3 space-y-2 lg:py-3 hover:bg-surface-25">
-                  <BodyText size={2}>Starting from</BodyText>
-                  <DatePicker
-                    dateTime={startDateTime}
-                    setDateTime={handleStartDateTimeChange}
-                    timeCaption="Start time"
-                    className="w-full"
-                  />
-                </div>
-                <div
-                  className={cx(
-                    "flex flex-col w-full pl-4 pr-3 py-2 lg:py-3 space-y-2 hover:bg-surface-25",
-                    {
-                      "!border !border-danger-200 !rounded-r-2xl":
-                        showPastEndDateError,
-                    }
-                  )}
-                >
-                  <BodyText size={2}>Until</BodyText>
-                  <DatePicker
-                    dateTime={endDateTime}
-                    setDateTime={(date: Date) => {
-                      const isEndTimeBeforeStartTime =
-                        date.getTime() <= startDateTime.getTime();
-
-                      setShowPastEndDateError(isEndTimeBeforeStartTime);
-                      deselectStrategy();
-                      setEndDateTime(date);
-                    }}
-                    timeCaption="End time"
-                    className="w-full"
-                    fromDate={startDateTime}
-                  />
+            <Checkbox
+              id="display-custom-date-range"
```

Comment:
I would add a `checkbox-freq` or just `checkbox` prefix to make it clear what the ID is.

```suggestion
              id="checkbox-freq-display-custom-date-range"
```